keyboards/crkbd/keymaps/blipson/keymap.c

```diff
@@ -0,0 +1,139 @@
+       KC_LCTL,   A(KC_F12),   A(G(KC_LEFT)), A(G(KC_RIGHT)), S(KC_F6),  C(S(KC_D)),                       C(S(KC_R)),  G(KC_LBRC),  G(KC_RBRC),  KC_F11,  KC_F12, KC_MPLY,
+   //|----------+------------+--------------+---------------+-----------+------------|                    |------------+------------+------------+--------+--------+---------|
+       G(KC_F2), A(G(KC_F)),  KC_MUTE,       KC__VOLDOWN,     KC__VOLUP, G(KC_GRV),                        C(G(KC_G)),  KC_TAB,      S(KC_TAB),   KC_BRID, KC_BRIU, KC_CAPS,
+   //|----------+------------+--------------+---------------+-----------+------------+--------|  |--------+------------+------------+------------+--------+--------+---------|
+                                                             _______,    KC_LGUI,     KC_SPC,     KC_SPC,   KC_RALT,     _______
+                                                         //`----------------------------------'  `-------------------------------'
+   )
+ };
+
+#ifdef OLED_ENABLE
+oled_rotation_t oled_init_user(oled_rotation_t rotation) {
+  if (!is_keyboard_master()) {
+    return OLED_ROTATION_180;  // flips the display 180 degrees if offhand
+  }
+  return rotation;
+}
+
+#define L_BASE 0
+#define L_LOWER 2
+#define L_RAISE 4
+#define L_ADJUST 8
+
+void oled_render_layer_state(void) {
+    oled_write_P(PSTR("Layer: "), false);
+    switch (layer_state) {
+        case L_BASE:
+            oled_write_ln_P(PSTR("Default"), false);
+            break;
+        case L_LOWER:
+            oled_write_ln_P(PSTR("Lower"), false);
+            break;
+        case L_RAISE:
+            oled_write_ln_P(PSTR("Raise"), false);
+            break;
+        case L_ADJUST:
+        case L_ADJUST|L_LOWER:
+        case L_ADJUST|L_RAISE:
+        case L_ADJUST|L_LOWER|L_RAISE:
+            oled_write_ln_P(PSTR("Adjust"), false);
+            break;
+    }
+    oled_write_P(PSTR("I love Marce"), false);
+}
+
+void render_bootmagic_status(bool status) {
+    /* Show Ctrl-Gui Swap options */
+    static const char PROGMEM logo[][2][3] = {
+        {{0x97, 0x98, 0}, {0xb7, 0xb8, 0}},
+        {{0x95, 0x96, 0}, {0xb5, 0xb6, 0}},
+    };
+    if (status) {
+        oled_write_ln_P(logo[0][0], false);
+        oled_write_ln_P(logo[0][1], false);
+    } else {
+        oled_write_ln_P(logo[1][0], false);
+        oled_write_ln_P(logo[1][1], false);
+    }
+}
+
+void oled_render_logo(void) {
+    static const char PROGMEM crkbd_logo[] = {
+        0x80, 0x81, 0x82, 0x83, 0x84, 0x85, 0x86, 0x87, 0x88, 0x89, 0x8a, 0x8b, 0x8c, 0x8d, 0x8e, 0x8f, 0x90, 0x91, 0x92, 0x93, 0x94,
+        0xa0, 0xa1, 0xa2, 0xa3, 0xa4, 0xa5, 0xa6, 0xa7, 0xa8, 0xa9, 0xaa, 0xab, 0xac, 0xad, 0xae, 0xaf, 0xb0, 0xb1, 0xb2, 0xb3, 0xb4,
+        0xc0, 0xc1, 0xc2, 0xc3, 0xc4, 0xc5, 0xc6, 0xc7, 0xc8, 0xc9, 0xca, 0xcb, 0xcc, 0xcd, 0xce, 0xcf, 0xd0, 0xd1, 0xd2, 0xd3, 0xd4,
+        0};
+    oled_write_P(crkbd_logo, false);
+}
+
+void oled_task_user(void) {
+    if (is_keyboard_master()) {
```

Comment:
Just a heads up, you could do `is_keyboard_left()` here instead, if you add `#define SPLIT_LAYER_STATE_ENABLE` to your config.h, as that will sync the layer state to both sides.  

This would allow you to display the layer state on either side.